notebook/cuIndicator/indicator_demo.ipynb

```diff
@@ -96,15 +96,15 @@
   },
   {
```

Comment:
Cell 4 with `get_figure` function is massive. Can that be refactored into a [factory ](https://realpython.com/factory-method-python/) and into a separate module? Then import from the module.

Reply:
It is a different task. Let's postpone this for the future.